### ISSUE TITLE
Xcode 9 Coverage

### ISSUE THREAD

**Issue by bamx23** [openbakery/gradle-xcodePlugin#371]
There is a problem with coverage report generation and Xcode 9.
Xcode 8 stores coverage data in `build/derivedData/Build/Intermediates/CodeCoverage/`, but Xcode 9 plases it into `build/derivedData/Build/ProfileData/<SOME_GUID>/`.

**Comment by phatblat** [comment]
I noticed the coverage task not picking up the coverage data from some Xcode 9 projects and captured some notes but didn't get around to creating an issue. Thanks for doing that @bamx23 

### Xcode 8

- iOSApp/build/derivedData/Build/Intermediates/CodeCoverage/Coverage.profdata
- Example/build/derivedData/Build/Intermediates/CodeCoverage/Coverage.profdata

### Xcode 9

- Example/build/derivedData/Build/ProfileData/68050F37-350D-4C3A-B1B9-6A427974C855/Coverage.profdata
- MDKApp/build/derivedData/Build/ProfileData/68050F37-350D-4C3A-B1B9-6A427974C855/Coverage.profdata
- test-iOSApp/build/derivedData/Build/ProfileData/68050F37-350D-4C3A-B1B9-6A427974C855/Coverage.profdata
- SignInDemo/build/derivedData/Build/ProfileData/68050F37-350D-4C3A-B1B9-6A427974C855/Coverage.profdata

The GUID in the Xcode 9 paths appears to be a simulator identifier:

```
 xcrun simctl list | grep 68050F37-350D-4C3A-B1B9-6A427974C855
    iPhone 6s (68050F37-350D-4C3A-B1B9-6A427974C855) (Shutdown)
```

Now, the question is if the plugin can determine the simulator GUID and pass that to the coverage task.

**Comment by phatblat** [comment]
Can this be closed now that #381 has been merged?

**Comment by bamx23** [comment]
Oh, sure! Thank you!
